makefile

```diff
@@ -90,3 +90,12 @@ mkdir-output:
 .PHONY: clean
 clean:
 	$(RM) $(OUTPUT_DIR)
+	
+# run checks
+checks: dna; \
+	cp $(OUTPUT_DIR)/${JARFILE} $(R_DIR)/inst/extdata/; \
+	make rDNA; \
+	rm $(R_DIR)/inst/extdata/${JARFILE}; \
+	cd $(OUTPUT_DIR); \
+	R CMD check --as-cran *.tar.gz; 
```

Comment:
Can you also clean up and remove the directories the check has created? Perhaps here and/or under `clean:` above?

Reply:
The checks run in `$(OUTPUT_DIR)` which is also where the folder `rDNA.Rcheck` is created. I was looking for a way to delete it when no errors or notes are returned but otherwise I think it makes sense to keep the folder in case you want to look at the logs. `$(OUTPUT_DIR)` is removed in `clean` already.

Reply:
OK, great!